frontend/website/posts/testnet-retro-5.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: Coda Protocol Testnet [Beta] Retrospective — Week 5
+date: 2019-08-27
+author: Christine Yip & Pranay Mohan
+---
+
+Welcome to week 6 of the Coda Public Testnet Beta. This week’s challenge, Something Snarky, is all about generating zk-SNARKs. Node operators will be competing to produce as many SNARKs as possible to sell to block producers on the snarketplace — read on for more details. In addition, we have a brand new [leaderboard](https://codaprotocol.com/testnet.html) design to keep track of your place on the list.
+
+![](https://miro.medium.com/max/3520/1*oe8TP8e3kBp2jFkYowCiKw.jpeg)
```

Comment:
Actually can you double check that these image links are right? This one doesn't look like the one on medium